tutor_webhook_receiver/patches/k8s-deployments

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webhook_receiver
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: webhook_receiver
+  template:
+    metadata:
+      labels:
+        app: webhook_receiver
+    spec:
+      containers:
+        - name: webhook_receiver
+          image: {{ DOCKER_REGISTRY }}{{ WEBHOOK_RECEIVER_DOCKER_IMAGE }}
+          ports:
+            - containerPort: 8090
+          env:
+            - name: DB_MIGRATION_ENGINE
+              value: django.db.backends.mysql
+            - name: DB_MIGRATION_NAME
+              value: { { WEBHOOK_RECEIVER_DB_NAME } }
```

Comment:
Ooof, missed this. This can't work; you can't have spaces between braces `{ {` and `} }`. Send a follow-up PR to fix this please?

Reply:
I have no idea how a simple copy-paste can fail in an IDE!! I will fix it

Reply:
Fix included in https://github.com/hastexo/tutor-contrib-webhook-receiver/pull/7.